### ISSUE TITLE
Upgrading TFS also needs a re-install of the aggregator

### ISSUE THREAD

**Issue by renevanosnabrugge** [tfsaggregator/tfsaggregator#112]
When I upgraded TFS from 2013.4 to 2015.1 the aggregator needed to be re-installed. This makes sense. But after re-installing the aggregator it did not work. Only after I uninstalled the aggregator (manually from the plugins folder) and with the MSI wizard, and re-installed the aggregator again it worked again.

I think it would be very convenient that when you install the aggregator with the MSI it does the following things:
- [ ] Find the newest location of the TFS plugins foler (2015.1 in my case)
- [ ] Copy the policies from the old location to the new location
- [ ] Uninstall the old aggregator and remove all the files
- [ ] Install the new aggregator

That would be very convenient!

Thanks for the great work !

**Comment by jessehouwing** [comment]
IIS reset is automatically detected by IIS because we change files in the bin directory.

**Comment by giuliov** [comment]
It would be very nice to have the install logs (`msixec /i ` _msipath_ `/Lv*x` _logpath_; `/x` to uninstall) but the installer code look up in the registry for the installed TFS version at install time only. I suspect that we should add more code to handle the scenario of TFS upgrade.
Thanks Jesse for upgrading the docs.

**Comment by giuliov** [comment]
Add a check for mismatch between Aggregator and TFS.
